README.md

```diff
@@ -1,15 +1,88 @@
+# Quickstart
 
-# Working on the operator
+## Prerequisites
 
-You can compile the operator image running:
+Submariner has a few requirements to get started:
+
+- At least 2 Kubernetes clusters, one of which is designated to serve as the central broker that is accessible by all of your connected clusters; this can be one of your connected clusters, but comes with the limitation that the cluster is required to be up to facilitate interconnectivity/negotiation
+- Different cluster/service CIDR's (as well as different Kubernetes DNS suffixes) between clusters. This is to prevent traffic selector/policy/routing conflicts.
+- Direct IP connectivity between the gateway nodes through the internet (or on the same network if not running Submariner over the internet). Submariner supports 1:1 NAT setups but has a few caveats/provider-specific configuration instructions in this configuration.
+- Knowledge of each cluster's network configuration
+- Worker node IPs on all the clusters must be outside of the cluster/service CIDR ranges.
+
+An example of three clusters configured to use with Submariner would look like the following:
+
+| Cluster Name | Provider | Cluster CIDR | Service CIDR | DNS Suffix |
+|:-------------|:---------|:-------------|:-------------|:-----------|
+| broker       | AWS      | 10.42.0.0/16 | 10.43.0.0/16 | cluster.local |
+| west         | vSphere  | 10.0.0.0/16  | 10.1.0.0/16  | west.local |
+| east         | OnPrem   | 10.98.0.0/16 | 10.99.0.0/16 | east.local |
+
+## Installation
+
+### Download
+Download the latest binary for your operating system and architecture, then install it on your path. The binary has no external dependencies.
+
+To install for Linux amd64 in `~/.local/bin`:
+
+    mkdir -p ~/.local/bin
+    wget https://github.com/submariner-io/submariner-operator/releases/download/v0.1.0/subctl-v0.1.0-linux-amd64
+    install subctl-v0.1.0 -linux-amd64 ~/.local/bin/subctl
+    rm subctl-v0.1.0-linux-amd64
+If `~/.local/bin` is on your PATH, you will then be able to run subctl.
+
+## Deployment
+
+### Submariner
+
+To deploy submariner in a two Kubernetes cluster setup follow the below steps
+
+* Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster run
+```
+subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane
+```
+
+
+* Join Clusters with Broker - To join all the other clusters with the broker cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
+
+```
+subctl join --kubeconfig <PATH-TO-KUBECONFIG-DATA-CLUSTER> broker-info.subm
+```
+
+### Submariner with Service Discovery
+
+ **Project status**: The project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features.
+
+The [Lighthouse](https://github.com/submariner-io/lighthouse) helps in cross-cluster service discovery. It has the below additional dependencies
```

Comment:
nit: The Lighthouse "project" helps ....

Reply:
Done.